### PR TITLE
Add English version

### PR DESCRIPTION
**Pull request by seratch**
This is a draft PR to add English version to this app.

### CODE REVIEW

File: app/tutorials.py [hunk 1]
```diff
@@ -82,13 +115,22 @@
             "type": "section",
             "text": {
                 "type": "mrkdwn",
-                "text": """
+                "text": i18n(
+                    """
+Using <https://api.slack.com/events-api|*Events API*> is the common way to set up and maintain Home tab content. The <https://api.slack.com/events/app_home_opened|*app_home_opened*> event triggers when an end user access Home Tab. Your app can subscribe the event and update the tab for the user. Actually, the event triggerred when you accessed this tab for the first time and this app updated the tab quickly.
```
Comment (mwbrooks) [inline app/tutorials.py]:
```suggestion
Using <https://api.slack.com/events-api|*Events API*> is the common way to set up and maintain Home tab content. The <https://api.slack.com/events/app_home_opened|*app_home_opened*> event triggers when an end user access Home Tab. Your app can subscribe the event and update the tab for the user. Actually, the event triggered when you accessed this tab for the first time and this app updated the tab quickly.
```

File: app/tutorials.py [hunk 2]
```diff
@@ -82,13 +115,22 @@
...
+
+It's also possible to update Home tabs at any timing apart from end users' access. For instance, your app can update tabs as part of midnight batch processes and/or asking end users to manually click buttons to refresh the contents. In this tutorial app, this app never updates unless you click *Previus* or *Next* after the initial loading.
```
Comment (mwbrooks) [inline app/tutorials.py]:
```suggestion
It's also possible to update Home tabs at any time regardless of the end users' access. For instance, your app can update tabs as part of midnight batch processes and/or asking end users to manually click buttons to refresh the contents. In this tutorial app, this app never updates unless you click *Previous* or *Next* after the initial loading.
```

File: app/tutorials.py [hunk 3]
```diff
@@ -193,14 +289,49 @@ def handle_modals(ack, view):
...
+@app.options("demo-selector")
+def external_data_source_handler(ack, body):
+    keyword = body.get("value")
+    if keyword is not None and len(keyword) > 0:
+        options = [o for o in all_options if keyword in o["text"]["text"]]
+        ack(options=options)
+    else:
+        ack(options=all_options)
+```
+As you see, the above code have a static array of options. But needless to say, your app can load data from anywhere. Apps can run queries to fetch data in database and/or talk to any backend services. While we used the select menu in Home tabs, the component is available for messages and modals too.
```
Comment (mwbrooks) [inline app/tutorials.py]:
```suggestion
As you see, the above code has a static array of options. But needless to say, your app can load data from anywhere. Apps can run queries to fetch data in database and/or talk to any backend services. While we used the select menu in Home tabs, the component is available for messages and modals too.
```

File: app/tutorials.py [hunk 4]
```diff
@@ -193,14 +289,49 @@ def handle_modals(ack, view):
...
+
+This is a greatly useful feature for business operations. Make use of it in many situations!
```
Comment (mwbrooks) [inline app/tutorials.py]:
```suggestion
This is a very useful feature for business operations. Make use of it in many situations!
```

File: app/tutorials.py [hunk 5]
```diff
@@ -295,13 +456,44 @@ def external_data_source_handler(ack, body):
...
+
+Among the Web APIs that are available at <https://api.slack.com/methods|api.slack.com/methods>, the ones having the admin. prefix are available for Enterprise Grid admin users. Organization admins can use the admin.* APIs with their user tokens for running management operations via APIs.
+
+*SCIM API*
+
+<http://www.simplecloud.info/|SCIM (System for Cross-domain Identity Management)> is a public specification that is supported by many services. <https://api.slack.com/scim|SCIM API> is available for Plus with SSO and Enterprise Grid plans.
+
+*Audit Logs API*
+
+<https://api.slack.com/admins/audit-logs|Audit Logs API> is available for monitoring various audit events in an Enterprise Grid organization. See the<https://api.slack.com/admins/audit-logs|document> to learn how to use it.
```
Comment (mwbrooks) [inline app/tutorials.py]:
```suggestion
<https://api.slack.com/admins/audit-logs|Audit Logs API> is available for monitoring various audit events in an Enterprise Grid organization. See the <https://api.slack.com/admins/audit-logs|document> to learn how to use it.
```

File: app/tutorials.py [hunk 6]
```diff
@@ -351,13 +551,43 @@ def external_data_source_handler(ack, body):
...
+            },
         },
         {
             "type": "section",
             "text": {
                 "type": "mrkdwn",
-                "text": """
+                "text": i18n(
+                    """
+Lastly, here are helpful resource to learn further about interactive features in the Slack Platform,.
```
Comment (mwbrooks) [inline app/tutorials.py]:
```suggestion
Lastly, here are helpful resource to learn further about interactive features in the Slack Platform:
```

**Review by mwbrooks — COMMENTED** [review]
Incredible app @seratch! While using the app, I learned a few new things as well. For example, I didn't know that you could update the App Home screen at any time, regardless of whether the user is accessing it.

I tried to run through all of the functionality and links. It all looks good to me!

**Comment by seratch** [comment]
@mwbrooks Thanks for reviewing this! also, I'm happy to learn you've learned some from this!

**Comment by seratch** [comment]
The English version has been updated with your improvements - thanks! https://j.mp/slack-dev-en

**Pull request merged**